python/setup.py

```diff
@@ -5,14 +5,29 @@ def load_dependencies():
         required = f.read().splitlines()
     return required
 
+
+def getReadMe():
+    with open("README.md", "r") as f:
+        long_description = f.read()
+    return long_description
+
 setup(
     name="ml4ir",
     packages=find_namespace_packages(include=["ml4ir.*"]),
     version="0.0.1",
     description="Machine Learning libraries for Information Retrieval",
+    long_description=getReadMe(),
+    long_description_content_type="text/markdown",
     author="Search Relevance, Salesforce",
-    author_email="searchrelevancyscrumteam@salesforce.com ",
-    data_files=[("ml4ir/build", ["ml4ir/build/Dockerfile"])],
+    author_email="searchrelevancyscrumteam@salesforce.com",
+    url="https://www.salesforce.com/",
+    classifiers=[
```

Comment:
are you following a tutorial? Have you done this in the past?
Looks good to me, but I have not release at pip

Reply:
Yes, I am following https://packaging.python.org/tutorials/packaging-projects/